### PR TITLE
Detach alias test for memory, disk, interface and rng device

### PR DESCRIPTION
**Pull request by kylazhang**
depends on 
https://github.com/avocado-framework/avocado-vt/pull/1724 
https://github.com/avocado-framework/avocado-vt/pull/1723 
https://github.com/avocado-framework/avocado-vt/pull/1722
Above all merged

new depends on
https://github.com/avocado-framework/avocado-vt/pull/2222
Also merged

### CODE REVIEW

**Comment by dzhengfy** [comment]
@kylazhang please help correct the checking error.

**Comment by kylazhang** [comment]
avocado-framework/avocado-vt#2222

File: libvirt/tests/src/libvirt_mem.py [hunk 1]
```diff
@@ -179,6 +180,8 @@ def check_dom_xml(at_mem=False, dt_mem=False):
             assert int(max_mem_rt) == xml_max_mem_rt
 
             # Check attached/detached memory
+            logging.info("at_mem=%s,dt_mem=%s" % (at_mem, dt_mem))
```
Comment (dzhengfy) [inline libvirt/tests/src/libvirt_mem.py]: logging.info("at_mem=%s,dt_mem=%s" , at_mem, dt_mem)

File: libvirt/tests/src/libvirt_mem.py [hunk 2]
```diff
@@ -179,6 +180,8 @@ def check_dom_xml(at_mem=False, dt_mem=False):
...
+            logging.info("detach_device is %s" % detach_device)
```
Comment (dzhengfy) [inline libvirt/tests/src/libvirt_mem.py]: same as above

File: libvirt/tests/src/libvirt_mem.py (hunk 1 above)
Comment (kylazhang) [reply libvirt/tests/src/libvirt_mem.py]: fixed, thanks

File: libvirt/tests/src/libvirt_mem.py (hunk 2 above)
Comment (kylazhang) [reply libvirt/tests/src/libvirt_mem.py]: fixed, thanks

**Pull request merged**